docker/docker-compose-build.yml

```diff
@@ -10,7 +10,7 @@ services:
       - "19002:3306"
 
   bind9:
-    image: "vinyldns/bind9:0.0.1"
+    image: "vinyldns/bind9:0.0.4"
```

Comment:
change this to `0.0.5`

Also, set the vinyldns version to `0.9.3` - that is important as we certify this release of the client works with this specific vinyldns version